core-api/src/main/java/org/silverpeas/core/ui/DisplayI18NTranslationMap.java

```diff
@@ -0,0 +1,105 @@
+/*
+ * Copyright (C) 2000 - 2022 Silverpeas
+ *
+ * This program is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU Affero General Public License as
+ * published by the Free Software Foundation, either version 3 of the
+ * License, or (at your option) any later version.
+ *
+ * As a special exception to the terms and conditions of version 3.0 of
+ * the GPL, you may redistribute this Program in connection with Free/Libre
+ * Open Source Software ("FLOSS") applications as described in Silverpeas's
+ * FLOSS exception.  You should have received a copy of the text describing
+ * the FLOSS exception, and it is also available here:
+ * "https://www.silverpeas.org/legal/floss_exception.html"
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU Affero General Public License for more details.
+ *
+ * You should have received a copy of the GNU Affero General Public License
+ * along with this program.  If not, see <http://www.gnu.org/licenses/>.
+ */
+
+package org.silverpeas.core.ui;
+
+import org.silverpeas.core.util.StringUtil;
+
+import java.util.HashMap;
+import java.util.Map;
+import java.util.Optional;
+
+import static org.silverpeas.core.util.StringUtil.EMPTY;
+import static org.silverpeas.core.util.StringUtil.isDefined;
+
+/**
+ * This map is dedicated to simple translation management.
+ * @author silveryocha
+ */
+public class DisplayI18NTranslationMap extends HashMap<String, String> {
```

Comment:
I wanted to keep the difference it has today between DisplayI18nHelper (user languages) and I18NHelper (implementation of I18n API for content languages)

Reply:
Ok. I think in fact `DisplayI18n` prefix is very bad named because in fact it has nothing to do with the action of displaying. It should be better to be `Useri18n`. But this is another story (unless you are ok with me and you're ok to rename them :-P )

Reply:
I can rename the new `DisplayI18NTanslationMap` class into `UserI18NTanslationMap` if you want.
Do you?

Reply:
Yes, it would be nice. `DisplayI18nHelper` should be also renamed to `UserI18nHelper` ... what do you think about this opportunity?